traitsui/editors/api.py

```diff
@@ -7,11 +7,15 @@
 try:
     from .array_editor import ArrayEditor
 except ImportError as e:
-    if 'numpy' not in e.message:
```

Comment:
If everything goes cleanly, this puts `numpy` into the namespace of the module, which we don't want for an "api" module.

We could `import numpy as _numpy` or something, I guess.